README.md

```diff
@@ -24,6 +24,8 @@ Open another terminal, start autocompiling smart-contracts
 
 ```bash
 lein auto compile-solidity
+# or alternatively use (note: truffle is currently experimental)
+truffle compile
```

Comment:
Starting dev server in this workflow, on `node dev-server/name-bazaar.js` I get `could not start [#'district.server.smart-contracts/smart-contracts] due to Error: ENOENT: no such file or directory, open '/home/ladislav/prj/NameBazaar/name-bazaar/resources/public/contracts/build/AuctionOfferingFactory.abi'`, and indeed the old directory doesn't contain build now.

Reply:
Oh, my bad. Yeah, currently truffle compile is not a replacement for `lein compile`.

It compiles the contracts, but only the "truffle way" and the clojure(script) layer needs the ABI's